packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -42,6 +44,7 @@ export interface MonitoringParams {
   unknownProposalNotificationInterval: number;
   retryAttempts: number;
   retryDelayMs: number;
+  checkBeforeExpirationSeconds: number;
```

Comment:
i think this makes sense but just want to double check, when specifying this parameter, this is how much time we need before dispute period ends?

Reply:
Yes, that’s exactly right. The parameter specifies the number of seconds remaining until the dispute period ends so we only process proposals when there are fewer than that many seconds left before expiration.